<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng Nhập</string>
  </property>
  <property name="styleSheet">
   <string>background-color: #F0F4F8;</string>
  </property>
  <!-- Tiêu đề -->
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Đăng Nhập</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string>background-color: #81D4FA; color: white; border-radius: 5px;</string>
   </property>
  </widget>
  <!-- Tài khoản -->
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tài khoản:</string>
   </property>
   <property name="styleSheet">
    <string>color: #333333;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: #FFFFFF; border: 1px solid #E67E22; border-radius: 3px; padding: 2px; color: #333333;</string>
   </property>
  </widget>
  <!-- Mật khẩu -->
  <widget class="QLabel" name="label_pass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Mật khẩu:</string>
   </property>
   <property name="styleSheet">
    <string>color: #333333;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_password">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="styleSheet">
    <string>background-color: #FFFFFF; border: 1px solid #E67E22; border-radius: 3px; padding: 2px; color: #333333;</string>
   </property>
  </widget>
  <!-- Nút Đăng Nhập -->
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>160</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Đăng Nhập</string>
   </property>
   <property name="styleSheet">
    <string>background-color: #F5CBA7; color: white; border-radius: 5px; font-size: 14px;</string>
   </property>
  </widget>
  <!-- Nút Quên mật khẩu và Đăng ký -->
  <widget class="QPushButton" name="btn_forgot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Quên mật khẩu?</string>
   </property>
   <property name="styleSheet">
    <string>background-color: #3498DB; color: white; border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>210</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Đăng ký</string>
   </property>
   <property name="styleSheet">
    <string>background-color: #FF6F61; color: white; border-radius: 5px;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>